### ISSUE TITLE
Can't find variable: name in NavBar

### ISSUE THREAD

**Issue by mamkkl** [galio-org/galio#128]
Exception when using NavBar, exception message as below:

```
ReferenceError: Can't find variable: name

This error is located at:
  in NavBar (at theme/index.js:33)
  in _class (at LoginComponent.js:30)
  in RCTView (at View:.js 35)
  in View (at SafeAreaView.js:40)
  ...
  ...
```

Located in NavBar.js:47 
`if (name || back) {`

My env
react: 16.8.6
react-native: 0.60.5
galio: 0.6.0

**Comment by palingheorghe** [comment]
Hi @mamkkl ,

Change the `name` variable to `leftIconName`. This should solve the problem locally. I'll make a hotfix for this issue as soon as possible.
🧜🏽‍♂️

**Comment by mamkkl** [comment]
Hi @palingheorghe ,

I can run the app by change name to leftIconName. I will create a pull request for it.

Thank you.

**Comment by palingheorghe** [comment]
Closed this because I fixed this with a merge in v0.6.1, you can find out more about this hotfix here: https://github.com/galio-org/galio/releases/tag/v0.6.1
